src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -547,3 +547,16 @@ g.test('reverse_depth')
       }
     );
   });
+
+g.test('create_pipeline_with_all_formats')
+  .desc('Test that creating a render pipeline works with for all depth and combined formats.')
+  .params(u =>
+    u
+      .combine('depthStencilFormat', kDepthStencilFormats)
+      .filter(({ depthStencilFormat }) => depthStencilFormat !== 'depth32float-stencil8')
```

Comment:
It looks like the 'depth32float-stencil8' format can't create a render pipeline yet.

Reply:
It can be supported, it just requires a device feature flag to be enabled. Pushed a commit fixing this.